packages/cursorless-engine/src/scopeProviders/ScopeInfoProvider.ts

```diff
@@ -176,6 +176,7 @@ function isLanguageSpecific(scopeType: ScopeType): boolean {
       return false;
 
     case "oneOf":
+    case "contiguous":
```

Comment:
can't you just call this function on the constituent scope?

Reply:
fixed